server/clearbit.js

```diff
@@ -154,32 +164,32 @@ export default class Clearbit {
     const domain = getDomain(user);
 
     if (!this.client || !discover_enabled || _.isEmpty(discover_segments)) {
-      this.debug("Discover not enabled", { discover_segments });
+      this.logSkip(user, "discover", "Discover not enabled", { discover_segments });
```

Comment:
@michaloo yes, but yestarday @sbellity asked me to do this logSkip method so I changed it. I am a bit confused which convention should I pick ?